src/components/motion/MotionController.cpp

```diff
@@ -21,27 +23,26 @@ void MotionController::Update(int16_t x, int16_t y, int16_t z, uint32_t nbSteps)
   }
 }
 
-bool MotionController::Should_RaiseWake(bool isSleeping) {
-  if ((x + 335) <= 670 && z < 0) {
-    if (not isSleeping) {
-      if (y <= 0) {
-        return false;
-      } else {
-        lastYForWakeUp = 0;
-        return false;
-      }
-    }
+bool MotionController::ShouldRaiseWake() const {
+  constexpr int16_t xThresh = 512;
+  constexpr int16_t yThresh = 0;
+  constexpr int16_t ySwitchThresh = -768;
+  constexpr int16_t requiredSpeed = 256;
 
-    if (y >= 0) {
-      lastYForWakeUp = 0;
-      return false;
-    }
-    if (y + 230 < lastYForWakeUp) {
-      lastYForWakeUp = y;
-      return true;
-    }
+  if (x < -xThresh || x > xThresh || y > yThresh) {
+    return false;
   }
-  return false;
+
+  // Use z here because it's not possible to accurately detect movement between around y == -768 and y == -1024
+  // y can't be used in this range because it needs more movement for the values to change the same amount
```

Comment:
I'm not sure this is true. I set the Motion app range to +-2000 and couldn't see a difference in the amplitude of the detected motion whether the watch was placed horizontally or vertically. The BMA425 datasheet claims a typical non linearity of 0.5%FS, which I think means across the entire range.

Reply:
Again, this is gravity and not motion. This behaviour can be explained with simple trigonometry.
sin0 = 0
sin15 = 0.26
sin75 = 0.97
sin90 = 1

Reply:
Ah ok, then I shouldn't have put that in the comment then. The second point I make does still stand though I think, because if the watch moves from facing up with a y value of -800 for example to facing down with a y value of -800, you can't tell whether the wrist was moved or not.

Reply:
No, I think you're still thinking about gravity. If you have the watch in any orientation and quickly jerk it along Y, it will show a positive and a negative peak in acceleration, assuming it updated fast enough and the acceleration is within the operating range of the accelerometer, and then it will return to the same old value which is acceleration due to gravity.

Reply:
I think you're right, but that the MotionController doesn't update fast enough for that to be recorded in the deltaY variable. I also think that the flick to wake the device up is usually not going to be greater than the acceleration due to gravity. Because of this I think it makes most sense to assume that the acceleration values are all acceleration due to gravity.

Reply:
Thanks helping me get clarification on how the accelerometer works! I appreciate it.

I think that changing the polling rate won't affect this, as long as the SystemTask still runs at the same speed. I will go through my logic and make sure that it still holds up with my new knowledge.